README.md

```diff
@@ -168,30 +266,29 @@ drwxr-xr-x  13 kellyegreene  staff   442B Jun  2 11:02 .git
 -rw-r--r--   1 kellyegreene  staff    11K Jun  2 11:21 README.md
 ```
 
-Notice that at the top of the file output there are a bunch of files that start with a `.`, like
-`.DS_Store`
-
-Files like `.DS_Store` are not listed. That's because files that start with a `.` are hidden files.
-Your `.bash_profile` is a hidden file in your home directory.  If you want to see the hidden files you can add the `a` flag to `ls` by typing `$ ls -a`.
+Notice that at the top of the file output there are a bunch of files that start with
+a `.`, like `.DS_Store`
 
-If you're interested in where this convention came from check out [The history of hidden files](https://plus.google.com/101960720994009339267/posts/R58WgWwN9jp) .
+Files like `.DS_Store` are not listed. That's because files that start with a `.`
+are _hidden_ files. Your `.bash_profile` is a hidden file in your home directory.
+If you want to see the hidden files you can add the `a` flag to `ls` by typing `$ ls -a`.
 
-
-**Note:** *Combining flags is only valid for single-letter options. A "long option" such as*
-`--force` *is defined with more than one character and must be entered with its own flag.*
+**Note:** *Combining flags is only valid for single-letter options. A "long option"
```

Comment:
this doesn't make sense. "and can't be combined?"

Can it be broken down
--force
--remote
can't be --forcremote or --remoteforce 
?